CodeStarterCamp_Week1/PeperoFile.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  IceCream.swift
+//  CodeStarterCamp_Week1
+//
+//  Created by 이선준 on 2023/03/03.
+//
+
+// MARK: Step 3: 주문에 따른 빼빼로 그리기
+
+import Foundation
+
+func drawStickOfPepero(stickLen: Int) -> Void {
```

Comment:
`stickLen`이라는 변수명을 사용해주셨군요!
다만, API Design Guidelines의 [Use Terminology Well](https://www.swift.org/documentation/api-design-guidelines/#use-terminology-well)에 따르면 축약어 사용은 지양하라 지시하고 있습니다. 축약어 없는 네이밍으로 수정해볼까요?

한가지 더 말씀 드리자면, stick이라는 단어가 불필요하게 반복되는 느낌이 듭니다. [Promote Clear Usage](https://www.swift.org/documentation/api-design-guidelines/#promote-clear-usage) 파트를 읽어보시고 추가적으로 수정해볼까요?

마지막으로, 다른 함수도 마찬가지이지만 여기에만 코맨트 드립니다!
스위프트에서 반환 값이 없을 경우 Void는 생략하는게 일반적인 작성법입니다! (필요할 때는 적는 경우도 있긴 합니다만, 이 경우는 아니네요 😅)
```suggestion
func drawStickOfPepero(stickLen: Int) {
```